framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/EntityFrameworkCore/DistributedEvents/DbContextEventInbox.cs

```diff
@@ -64,23 +68,21 @@ public async Task MarkAsProcessedAsync(Guid id)
         }
 
         [UnitOfWork]
-        public async Task<bool> ExistsByMessageIdAsync(string messageId)
+        public virtual async Task<bool> ExistsByMessageIdAsync(string messageId)
         {
-            //TODO: Optimize
             var dbContext = await DbContextProvider.GetDbContextAsync();
             return await dbContext.IncomingEvents.AnyAsync(x => x.MessageId == messageId);
         }
 
         [UnitOfWork]
-        public async Task DeleteOldEventsAsync()
+        public virtual async Task DeleteOldEventsAsync()
         {
-            //TODO: Optimize
             var dbContext = await DbContextProvider.GetDbContextAsync();
-            var timeToKeepEvents = Clock.Now.AddHours(-2); //TODO: Config?
+            var timeToKeepEvents = Clock.Now.Add(DistributedEventsOptions.InboxKeepEventTimeSpan);
```

Comment:
It would be better to make `DistributedEventsOptions.InboxKeepEventTimeSpan` a positive value and subtract it here. Negative values can be confusing. Also `WaitTimeToDeleteProcessedInboxEvents` would be better than `InboxKeepEventTimeSpan`.